scripts/setup-centos8.sh

```diff
@@ -25,27 +37,32 @@ export CXXFLAGS=$CFLAGS  # Used by boost.
 export CPPFLAGS=$CFLAGS  # Used by LZO.
 CMAKE_BUILD_TYPE="${BUILD_TYPE:-Release}"
 BUILD_DUCKDB="${BUILD_DUCKDB:-true}"
+export CC=/opt/rh/gcc-toolset-9/root/bin/gcc
+export CXX=/opt/rh/gcc-toolset-9/root/bin/g++
 
 function dnf_install {
   dnf install -y -q --setopt=install_weak_deps=False "$@"
 }
 
-dnf update -y
-dnf_install epel-release dnf-plugins-core # For ccache, ninja
-dnf config-manager --set-enabled powertools
-dnf update -y
-dnf_install ninja-build cmake curl ccache gcc-toolset-9 git wget which libevent-devel \
-  openssl-devel re2-devel libzstd-devel lz4-devel double-conversion-devel \
-  libdwarf-devel curl-devel libicu-devel
-
-dnf_install autoconf automake libtool bison flex python3 libsodium-devel
+# Install packages required for build.
+function install_build_prerequisites {
+  dnf update -y
+  dnf_install epel-release dnf-plugins-core # For ccache, ninja
+  dnf config-manager --set-enabled powertools
+  dnf update -y
+  dnf_install ninja-build cmake curl ccache gcc-toolset-9 git wget which
+  dnf_install autoconf automake python3 libtool
```

Comment:
```suggestion
  dnf_install autoconf automake python39 python39-devel libtool
```
It will install 3.6 by default, -devel is for pyvelox

Reply:
Fixed